### PR TITLE
SOV-2083: add links to alpha

### PR DESCRIPTION
**Pull request by creed-victor**
https://sovryn.atlassian.net/browse/SOV-2083

### CODE REVIEW

File: apps/frontend/src/app/3_organisms/Header/components/ProductLinks/ProductLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+const linkToAlpha = (path?: string) =>
+  `${
+    isStaging()
+      ? ALPHA_LINKS.STAGING
+      : ALPHA_LINKS[isMainnet() ? Environments.Mainnet : Environments.Testnet]
+  }${path}`;
+
+const linkToBitocracy = isMainnet()
+  ? 'https://bitocracy.sovryn.app'
+  : 'https://bitocracy.test.sovryn.app';
```
Comment (tiltom) [inline apps/frontend/src/app/3_organisms/Header/components/ProductLinks/ProductLinks.tsx]: Extract these 2 links to `links.ts` (`BITOCRACY_LINKS`).

File: apps/frontend/src/app/3_organisms/Header/components/ProductLinks/ProductLinks.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+import { ReactComponent as AppsIcon } from '../../../../../assets/images/apps-icon.svg';
+import { ALPHA_LINKS } from '../../../../../constants/links';
+import { translations } from '../../../../../locales/i18n';
+import { Environments } from '../../../../../types/global';
+import { isMainnet, isStaging } from '../../../../../utils/helpers';
+import { ProductLink } from '../ProductLink/ProductLink';
+import styles from './ProductLinks.module.css';
+
+const linkToAlpha = (path?: string) =>
```
Comment (tiltom) [inline apps/frontend/src/app/3_organisms/Header/components/ProductLinks/ProductLinks.tsx]: Extract this logic (without path concatenation) to `helpers.ts` (similar to `D2_URL` in D1).

File: apps/frontend/src/app/3_organisms/Header/components/ProductLinks/ProductLinks.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  `${
+    isStaging()
+      ? ALPHA_LINKS.STAGING
+      : ALPHA_LINKS[isMainnet() ? Environments.Mainnet : Environments.Testnet]
+  }${path}`;
+
+const linkToBitocracy = isMainnet()
```
Comment (tiltom) [inline apps/frontend/src/app/3_organisms/Header/components/ProductLinks/ProductLinks.tsx]: Extract this logic (env detection) to `helpers.ts`.

**Review by tiltom — CHANGES_REQUESTED** [review]
Except the code issues, there are 2 issues that need to be fixed:
1. The D1 links in the mobile menu have completely different typography than regular links. Also, there's some margin/padding on the left. Make them consistent with the D1 links pls.
image

2. Remove "Sovryn Alpha" link from the Footer (it's in the spec).

**Review by soulBit — APPROVED** [review]

**Review by pietro-maximoff — APPROVED** [review]

**Pull request merged**